docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
-| background_refresh_metadata_time_secs_since_last_access_secs | s    | 86400  | The expiration time of a Hive metadata cache refresh task. For the Hive catalog that has been accessed, if it has not been accessed for more than the specified time, StarRocks stops refreshing its cahced metadata. For the Hive catalog that has not been accessed, StarRocks will not refresh its cached metadata. This parameter is supported from v2.5.5 onwards.                                                                                       |
+##### catalog_try_lock_timeout_ms
+
+- **Unit**: ms
+- **Default**: 5000
+- **Description**: The timeout duration to obtain the global lock.
+
+##### edit_log_roll_num
+
+- **Unit**: -
+- **Default**: 50000
+- **Description**: The maximum number of metadata log entries that can be written before a log file is created for these log entries. This parameter is used to control the size of log files. The new log file is written to the BDBJE database.
+
+##### ignore_unknown_log_id
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether to ignore an unknown log ID. When an FE is rolled back, the BEs of the earlier version may be unable to recognize some log IDs. If the value is `TRUE`, the FE ignores unknown log IDs. If the value is `FALSE`, the FE exits.
+
+##### ignore_materialized_view_error
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether FE ignores the metadata exception caused by materialized view errors. If FE fails to start due to the metadata exception caused by materialized view errors, you can set this parameter to `true` to allow FE to ignore the exception. This parameter is supported from v2.5.10 onwards.
+
+##### ignore_meta_check
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether non-leader FEs ignore the metadata gap from the leader FE. If the value is TRUE, non-leader FEs ignore the metadata gap from the leader FE and continue providing data reading services. This parameter ensures continuous data reading services even when you stop the leader FE for a long period of time. If the value is FALSE, non-leader FEs do not ignore the metadata gap from the leader FE and stop providing data reading services.
+
+##### meta_delay_toleration_second
+
+- **Unit**: s
+- **Default**: 300
+- **Description**: The maximum duration by which the metadata on the follower and observer FEs can lag behind that on the leader FE. Unit: seconds. If this duration is exceeded, the non-leader FE stops providing services.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/37abe88c-a38b-45f8-ae2b-13751843ab27)
-> the non-leader FEs stop
because we have follower and observer FEs in addition to one Leader FE in a cluster, and that solicits a plural form for "non-leader FEs".

Reply:
![image](https://github.com/StarRocks/starrocks/assets/57167462/23c39226-9c0f-49bd-97b9-52826ddd9e30)
"Unit: seconds." can be deleted, because the unit has been declared above in Line 85 "**Unit**: s".